style.css

```diff
@@ -0,0 +1,259 @@
+body {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  overflow-x: hidden;
+}
+
+/* Mobile Styles */
+@media screen and (min-width: 375px) {
+  .header-menu {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+  }
```

Comment:
## Styling ✨✨
- [ ] **Kudos on using the ```background-image``` 👏 👏** However, it would be better to replicate the [Figma design ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=wMKvp4oyMqZ6YudA-0) as much as possible by using the same background size and position 🙏 
- [ ] I think it would be better also to make some adjustments to the typography ```(font-size, font-weight, color)``` to replicate the chosen Figma design. Currently, your work are very small compared to that of the figma design  😎😎 
- [ ] It would be better also to replicate the spacing of your elements **(margin and padding)** given by the Figma design, 🙏 

   
  **_You may kindly try to match the following 👇_**
   ![image](https://user-images.githubusercontent.com/102757126/234969188-6ac29e16-9210-46b6-a57c-91ddc758558f.png)

Reply:
Changes have been implemented.Font and font weight have been increased.[](url)